src/main/java/com/rw/legion/columncheck/FloatChecker.java

```diff
@@ -37,12 +43,26 @@ public FloatChecker(JsonObject json) {
     
     public boolean validates(String str) {
         try {
-            if (floatType.equals("float")) Float.parseFloat(str);
-            if (floatType.equals("double")) Double.parseDouble(str);
+            if (floatType.equals("float")) {
+                Float value = Float.parseFloat(str);
+                if (Float.MAX_VALUE < value || Float.MIN_VALUE > value) {
```

Comment:
Why not just use `Float.isInfinite()`? https://docs.oracle.com/javase/7/docs/api/java/lang/Float.html

Reply:
I'll check that out

Reply:
This works for Floats and Doubles which are too large (because they equate to infinity), but numbers which are too small equate to the smallest valid Float/Double and thus falsely return true to `Float.isInfinite(value)`

Reply:
By "small" do you mean negative, or really, really close to zero?

Reply:
Really really close to 0. Exceeding their precision abilities.

`4.9E-325` for double
`1.4E-46` for float